src/commands/System/eval.js

```diff
@@ -1,4 +1,6 @@
-const { Command } = require('klasa');
+const now = require('performance-now');
```

Comment:
As stated in another comment, no more performance-now. This is the age of the Stopwatch